### ISSUE TITLE
Useful function (playlist autoplay off) removed

### ISSUE THREAD

**Issue by zer0701** [ParticleCore/Iridium#406]
tl;dr: please bring back the "playlist autoplay off" button. The "video autoplay" button is not fully redundant with it.

Long story:
Since the last update, the playlist autoplay function was removed. _Why_?? It was the single reason I wanted this addon for, the rest is just extra.

No, the "Autoplay" button _is not_ redundant with it, seeing as right now what happens is a video ends and the next one on the list is automatically loaded, albeit not automatically played.

What I like to do is to finish a video and then read the author's notes and the comments section, which the autoplay function worked _perfectly_ for. Right now, I have to pay attention to when the video is ending to pause it just as it's about to end, so I'm able to read through comments. But having to do that was the reason I wanted an addon in the first place!

**Comment by ParticleCore** [comment]
Duplicate of #78
